assemble-and-restart

```diff
@@ -71,6 +71,9 @@ if [ ! -z "${ODO_S2I_WORKING_DIR}" ] && [ -n  "$ODO_SRC_BACKUP_DIR" ] && ([ "${O
     rsync -rlO ${ODO_SRC_BACKUP_DIR}/src/. ${ODO_S2I_SRC_BIN_PATH}/src/
 fi
 
+# Now that the assemble is successful; change autostart from false to true in supervisor.conf present on persistent volume mounted on "/opt/app-root"
+PV_MNT_PT="/opt/app-root"
+sed -i 's/autostart=false/autostart=true/g' ${PV_MNT_PT}/conf/supervisor.conf
```

Comment:
`sed -i 's/autostart=false/autostart=true/g'` will change all the occurrences of `autostart=false`. At the moment, there's only one that we're adding as a part of this PR (correct me if I'm wrong) but if we add more in future, this will have unexpected consequences. Can we remove the `g` directive? Or if there's a better approach for this, then use that, maybe?

Reply:
@dharmit, `g` directive changes all the occurrences of the matched pattern on a **single line**. Even if `g` directive is removed, all the other occurrences of the matched pattern in the same file will still be changed (except multiple occurrence of the matched pattern on the same line).

And IMHO, the `autostart=false` pattern is very specific to supervisor.conf file.

Reply:
Even tho this is not the ideal way to do this, I agree with @kanchwala-yusuf, it is a config file where we know all possible values that can exist in it. It is good enough to use sed like this.